src/core_io.h

```diff
@@ -20,6 +20,12 @@ class uint256;
 class UniValue;
 class CTxUndo;
 
+enum class TxVerbosity {
+    DONT_SHOW_TX_DETAILS,
+    TX_DETAILS,
+    TX_DETAILS_AND_PREV_OUT
+};
```

Comment:
nit: I find something like this cleaner

```suggestion
enum class TxVerbosity {
    SHOW_TXID
    SHOW_DETAILS,
    SHOW_DETAILS_AND_PREV_OUT
};
```

Reply:
done, adopted change.

Question with code review like this, is it preferred to make a separate commit, or squash everything to a single commit ?

Reply:
For small code review like the above 'Suggested change' through the GitHub web interface it's best to just fix it up locally, squash and then force push IMO. No reason to have it a separate commit (and I personally don't care about the co-author thing that GitHub does).